test/engine/SpatialJoinTest.cpp

```diff
@@ -821,6 +823,66 @@ std::vector<std::vector<std::string>> expectedMaxDist10000000_rows_diff{
     mergeToRow(unordered_rows.at(4), unordered_rows_small.at(3),
                expectedDistEifLib)};
 
+std::vector<std::vector<std::string>> expectedNearestNeighbors1{
+    mergeToRow(unordered_rows.at(0), unordered_rows.at(0), expectedDistSelf),
+    mergeToRow(unordered_rows.at(1), unordered_rows.at(1), expectedDistSelf),
+    mergeToRow(unordered_rows.at(2), unordered_rows.at(2), expectedDistSelf),
+    mergeToRow(unordered_rows.at(3), unordered_rows.at(3), expectedDistSelf),
+    mergeToRow(unordered_rows.at(4), unordered_rows.at(4), expectedDistSelf)};
```

Comment:
Is it possible to create aliases for the `at(...)` calls, such that you can write something like

`mergeToRow(tf, eiffelTower, expectedDistTfEiffel)`.

That will make this much much easier to read.